src/pages/guides/uxp_guide/uxp-misc/manifest-v4/index.md

```diff
@@ -66,16 +66,16 @@ Key path | Type | Description | Required
 `name`   | `string` | The name should be 3 - 45 characters. We recommend your plugin name matches the _project name_ you created when getting your plugin ID from the Adobe Developer Console. | Develop / Publish
 `version`| `string` | Version number of your plugin in `x.y.z` format. <br/>Version must be three segments and each version component must be between `0` and `99`. | Develop / Publish
 `main`   | `string` | Path to the your plugin initialization code. This can be a JavaScript file or an HTML file. | Optional (defaults to `main.js`)
-`icons` | `array<object>` | Icons for your plugin (which may be rendered in various contexts, such as the plugin panel) <br/> PNG, JPG/JPEG formats are supported and the max file size for each icon is 1MB. <br/> You should specify at least the 1x and 2x size. Icons for the Plugin Manager are uploaded directly via the Adobe Developer Console, not included within your plugin itself. See our ["Publishing your plugin" guide](../../../../distribution/packaging-your-plugin/) to learn more. | Publish
-`host.app` | `string` | Indicates that this is a plugin for an Adobe app (currently, the only valid values here are `XD` or `PS`). | Develop / Publish
-`host.minVersion` | `string` | Minimum required version of the host app (in `x.y` format) that can run this plugin. The lowest valid version for Photoshop plugins is `22.0`. <br/>**Note:** The version number must be at least two segments. Typically, you'll leave the minor segment set to `0`, e.g. `22.0`. | Develop / Publish
-`host.maxVersion` | `string` | Maximum version of host app that can run this plugin. Same formatting as `host.minVersion`. | Optional
-`entryPoints` | `EntryPointDefinition]`| Describes the entries your plugin adds to the _Plugins_ menu & plugin panel. See the next section for details. | Develop / Publish
+`icons` | `array<IconDefinition>` | Icons for your plugin (which may be rendered in various contexts, such as the plugin panel) <br/> PNG, JPG/JPEG formats are supported and the max file size for each icon is 1MB. <br/> You should specify at least the 1x and 2x size. Icons for the Plugin Manager are uploaded directly via the Adobe Developer Console, not included within your plugin itself. See our ["Publishing your plugin" guide](../../../../distribution/packaging-your-plugin/) to learn more. | Publish
+| `host` | `array<HostDefinition>` | Describes the supported applications that can be used with this plugin. This can include the type of application, the minimum required version, or the maximum version of the host app that the plugin supports.  | Develop / Publish
```

Comment:
This is where we get into a little bit of confusion frequently -- during development UXP permits plugins to have multiple hosts in the manifest, but for publication to the marketplace, we currently restrict the plugin to one host. (INstead we'll be introducing "bundles" which let developers & users install plugins across apps). 

I think it's fine to have the array in the docs, but we'll want to make clear that the array is supported only during for development. Once a pluigin is going to be submitted to the marketplace, the dev needs to create a separate bundle that has one and only one host entry.

So we should update the type to `HostDefinition|HostDefinition[]` and add a note that for publishing, the non-array version must be used. The array itself is most useful for development.

Reply:
Interesting! I didn't know this was handled differently during development vs marketplace. I'll definitely make those changes and add a note to reflect this.

On a side note: just curious, what's the reasoning behind only allowing one host entry in a bundle?

Reply:
The primary reason is the disjointed release schedule of host applications -- which makes it hard to say _for sure_ that it's safe for a user to upgrade either a plugin or their host applications without breaking the plugin. For example, if the plugin says it works on the newest version of Ps & XD, and the user only updates XD & the plugin, then plugin _stops_ working on Ps.

For the marketplace, we'll us bundling to solve this, I think -- each part of the bundle can be updated independently, allowing the user to have one version of the plugin for Ps and another for XD.

Reply:
ah, that makes a lot of sense. Thanks for explaining it to me!